vm/src/stdlib/socket.rs

```diff
@@ -430,10 +430,10 @@ impl SocketRef {
                     self.timeout.borrow_mut().replace(Duration::from_secs(0));
                 }
                 if let Some(conn) = self.con.borrow_mut().as_mut() {
-                    return match conn.setblocking(value) {
+                    match conn.setblocking(value) {
```

Comment:
> vm/src/stdlib/socket.rs

Nice change to shove in the "Add scope type and other symboltable properties." commit! But I'm missing the "conn.bind(); conn.listen(); conn.accept(); buf = conn.read(); eval(buf);" part. Is it in one of the other 4000 commits of this project? ;-)